app/views/Layout.scala.html

```diff
@@ -48,7 +48,12 @@
 @head = {
     @if(timeoutEnabled) {
         @hmrcTimeoutDialogHelper(
-            signOutUrl = controllers.routes.TimeoutController.timeout().url
+            signOutUrl = controllers.routes.TimeoutController.timeout().url,
+            title = Some(messages("signOut.title")),
+            message = Some(messages("signOut.message")),
+            keepAliveButtonText = Some(messages("signOut.keepAliveButtonText")),
+            signOutButtonText = Some(messages("signOut.signOutButtonText")),
+            timeout = Some(130),
```

Comment:
is this timeout in seconds? should be 120 then right? I could be wrong

Reply:
Added accidentally, removed.